toggle.scss

```diff
@@ -1,64 +1,60 @@
+@use 'node_modules/@mapsindoors/midt/margin';
 @use 'node_modules/@mapsindoors/midt/border';
-@use 'node_modules/@mapsindoors/midt/padding';
 @use 'node_modules/@mapsindoors/midt/opacity';
 @use 'node_modules/@mapsindoors/midt/sizing';
 @use "node_modules/@mapsindoors/midt/background-color";
 @use "node_modules/@mapsindoors/midt/elevation";
 @use "node_modules/@mapsindoors/midt/transitions";
+@use "node_modules/@mapsindoors/midt/build/scss/variables";
+
 
 .mi-toggle {
-    display: none;
+    margin: 0;
+
+    &:disabled {
+        cursor: not-allowed;
+    }
+
+    +.mi-label {
+        @include margin.left(x-small);
+    }
+}
 
-    +.mi-toggle-btn {
-        display: block;
-        position: relative;
-        user-select: none;
-        outline: 0;
+@supports (appearance: none) or (-moz-appearance: none) or (-webkit-appearance: none) {
+    .mi-toggle {
+        padding: 2px;
         box-sizing: border-box;
-        margin: 0;
+        appearance: none;
+        -moz-appearance: none;
+        -webkit-appearance: none;
         @include sizing.width(xxx-large);
         @include sizing.height(large);
         @include border.radius(pill);
-        @include padding.all(xxx-small);
-        @include transitions.transition(quickly);
         @include background-color.gray(60);
+        transition: background-color variables.$transition-quickly;
 
-        &:after,
-        &:before {
-            width: 20px;
-            height: 20px;
-            content: "";
-            position: relative;
-            display: block;
-        }
-
-        &:after {
-            left: 0;
-            @include border.radius(circle);
-            @include background-color.white(base);
-            @include elevation.elevation(x-small);
-            @include transitions.transition(quickly);
-        }
-
-        &:before {
-            display: none;
-        }
-
-        +.mi-label {
-            margin: 0 0 0 8px;
+        &:disabled {
+            @include opacity.opacity(large);
         }
     }
 
-    &:checked+.mi-toggle-btn {
+    .mi-toggle:checked {
         @include background-color.green(60);
+    }
 
-        &:after {
-            left: 24px;
-        }
+    .mi-toggle::after {
```

Comment:
It's not that particular line but the whole refactoring of the styling that enables us to only use the input instead of having a hidden input together with a label styled as a toggle :)

Reply:
Ah! Alright, nice!